### ISSUE TITLE
MS Project import demo produces invalid calendar intervals

### ISSUE THREAD

**Issue by arcady-zherdev** [bryntum/support#5334]
The demo uses both static and recurrent start/end dates which is invalid.
```json
{
              "recurrentStartDate": "on FRIDAY at 00:00",
              "endDate": "2022-11-06T23:59:59",
              "priority": 25,
              "recurrentEndDate": "on SATURDAY at 00:00",
              "startDate": "2022-10-06T00:00:00",
              "isWorking": false
},
```

Reported here: https://www.bryntum.com/forum/viewtopic.php?f=52&t=22480

**Comment by mstfgueclue** [comment]
# Addition to the ticket

@ninjakon

**Comment by mstfgueclue** [comment]
# Addition to the ticket
My colleague @ninjakon has made the report. Additionally, we want to add some background information about the use case. 

## A rough overview of the workflow
  - The FE allows importing MS-Project files, which triggers the AWS Lambda function that uses the MPXJ library to read the project plans.
![image](https://user-images.githubusercontent.com/64474817/192980105-034003f5-7aac-461e-8825-b47ba4730c0a.png)

## Week Overrides
  - The screenshot below shows the debug result of the reading project, which highlights the JSON that is displayed in the description of @arcady-zherdev. This week's override with highlighted `startDate` and `endDate` should overwrite the standard interval by making Fridays to a non-working day. 
![image](https://user-images.githubusercontent.com/64474817/192980812-2cfb6cbc-5f2e-45de-b9de-8c4b0254de86.png)

> Thank you for keeping an eye on this!

**Comment by arcady-zherdev** [comment]
The issue is resolved in the upcoming 5.2.0 release
